src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -0,0 +1,477 @@
+package api.loans.scenarios;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.everyItem;
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.FeeAmount;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.hamcrest.Matcher;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.MultipleJsonRecords;
+import api.support.builders.CheckInByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+public class CheckInDeclaredLostItemTest extends APITests {
```

Comment:
Is there a test for what should happen when a fee has been partially paid? Should the paid part be refunded and the unpaid part be cancelled?

(similarly for transferred fees)

Reply:
Here it is `shouldCloseAndRefundPaidAndTransferredFee`.

When a fee has been partially paid/transferred then:
* refund paid/transferred part (by creating `credit` and `refund` actions);
* cancel remaining amount (by creating `cancellation` action);
* Update account: payment status - `Cancelled item returned`, status - `Closed`, remaining amount - `0.00`.

Reply:
Thanks, I think that answers one of my other questions, I'll resolve this comment